content/react-create-app.md

```diff
@@ -0,0 +1,13 @@
+# Getting started with Create React App
+
+### Theory
+1. [Create React App: Introduction](https://blog.logrocket.com/getting-started-with-create-react-app-d93147444a27/)
+1. [Create React App: Quick Start](https://create-react-app.dev/docs/getting-started#creating-an-app)
+
+### Practice
+
+>Hint: If you already set up and has launched the project,
+>you may found already prepared main file in: /src/App.js
+
+Make any changes within jsx(html) structure, try for example to change
+the text within the paragraph tag, and make sure that these changes are displayed in the browser.
```

Comment:
Need info about repository location, few steps to do (fork, create pull requests)